pkg/errors/errors.go

```diff
@@ -35,27 +36,53 @@ func init() {
 	}
 }
 
+func MakeErrID() string {
```

Comment:
Please rename to `makeErrID()`. No need to export the function.

Reply:
Done.